account_financial_risk/models/res_partner.py

```diff
@@ -215,11 +215,15 @@ def _compute_risk_exception(self):
                 max_value = getattr(partner, risk_field[1], 0.0)
                 if max_value and field_value > max_value:
                     partner.risk_exception = True
+                else:
+                    partner.risk_exception = False
                 if getattr(partner, risk_field[2], False):
                     amount += field_value
             partner.risk_total = amount
             if partner.credit_limit and amount > partner.credit_limit:
```

Comment:
Please do it more Pythonic :wink: :

```suggestion
                partner.risk_exception = partner.credit_limit and amount > partner.credit_limit
```